### PR TITLE
Publish only major versions: 1.x, 2.x etc.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: src/main/java/com/squareup/osstrich/JavadocPublisher.java [hunk 1]
```diff
@@ -116,4 +146,25 @@ private void gitCommitAndPush(String message) throws IOException {
     cli.withCwd(directory).exec("git", "commit", "-m", message);
     cli.withCwd(directory).exec("git", "push", "origin", "gh-pages");
   }
+
+  public static void main(String[] args) throws IOException {
```
Comment (swankjesse) [inline src/main/java/com/squareup/osstrich/JavadocPublisher.java]: Gradle!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
 # osstrich
-Maven plugin to help open source projects
+
+Open Source Software Task Robot Indestructible Computer Hero
```
Comment (JakeWharton) [inline README.md]: AMAZE

File: src/main/java/com/squareup/osstrich/JavadocPublisher.java [hunk 3]
```diff
@@ -116,4 +146,25 @@ private void gitCommitAndPush(String message) throws IOException {
...
+    Log log = new SystemStreamLog();
+
+    if (args.length != 3) {
+      log.info(String.format("Usage: %s <directory> <group ID> <repo URL>\n",
+          JavadocPublisher.class.getName()));
+      return;
```
Comment (JakeWharton) [inline src/main/java/com/squareup/osstrich/JavadocPublisher.java]: `System.exit(1)`

**Comment by JakeWharton** [comment]
LGTM

File: src/main/java/com/squareup/osstrich/JavadocPublisher.java (hunk 3 above)
Comment (swankjesse) [inline src/main/java/com/squareup/osstrich/JavadocPublisher.java]: Fixed.

**Pull request merged**